spec/index.bs

```diff
@@ -1662,14 +1783,23 @@ enum GPUTextureFormat {
 };
 </script>
 
-  * The `depth24plus` family of formats ({{GPUTextureFormat/depth24plus}} and
-    {{GPUTextureFormat/depth24plus-stencil8}})
-    must have a depth-component precision of
-    1 ULP &le; 1 / (2<sup>24</sup>).
+The following texture formats are considered <dfn dfn for="GPUTextureFormat">depth or stencil format</dfn>s:
+
+  - "depth32float"
+  - "depth24plus"
+  - "depth24plus-stencil8"
+
+There are no <dfn dfn for="GPUTextureFormat">compressed format</dfn>s defined in unextended WebGPU.
+Extensions may define some, though.
+
+The `depth24plus` family of formats ({{GPUTextureFormat/depth24plus}} and
+{{GPUTextureFormat/depth24plus-stencil8}})
+must have a depth-component precision of
+1 ULP &le; 1 / (2<sup>24</sup>).
 
-    Note: This is unlike the 24-bit unsigned normalized format family typically
-    found in native APIs, which has a precision of
-    1 ULP = 1 / (2<sup>24</sup> &minus; 1).
+Note: This is unlike the 24-bit unsigned normalized format family typically
+found in native APIs, which has a precision of
+1 ULP = 1 / (2<sup>24</sup> &minus; 1).
```

Comment:
I don't understand this part. Where do we see ULP of `2^23` with unsigned normalized formats?

Reply:
It's not 1/(2^23), it's 1/((2^24) - 1). Because 0x00_0000unorm = 0.0f, and 0xFF_FFFFunorm = 1.0f. Similar to how with colors, 0x00unorm = 0.0, 0xFFunorm = 1.0, so each step is 1/255.

Reply:
The important part of this callout is not the difference between 1/(2^24) and 1/(2^24 - 1), but that the unorm formats have a constant ULP, and the depth24plus formats may have a variable ULP with an upper bound. Could be clarified.

Reply:
Definitely needs to be clarified. Also, the claim that our `depth24plus` "must have a depth-component precision of ..." needs to be reconsidered. I think, the lower ULP is, the better for the user. So we can't claim our ULP being lower than the fixed-point ULP, since our depth24plus format can be backed by a native fixed-point D24X8 format.

Reply:
Oops, you're right, I did mess something up here. I thought I had determined that the precisions of these two formats crossed over at some point.

Reply:
I'll look into this again and submit a fix for it.